app/src/main/java/com/banuba/quickstart_c_api/GLNVRenderer.java

```diff
@@ -0,0 +1,295 @@
+package com.banuba.quickstart_c_api;
```

Comment:
rename the file to GLYUVNVRenderer, if you add the swap colors option then your renderer will support also NV21 format - it is the same as NV12 but bytes in UV plane are swapped, else please name file GLYUVNV12Renderer.